docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md

```diff
@@ -0,0 +1,110 @@
+---
+aliases:
+  - ../../../enterprise/activate-aws-marketplace-license/activate-license-on-ecs/
+  - ../../../enterprise/license/activate-aws-marketplace-license/activate-license-on-ecs/
+description: Configure Team LBAC rules for Loki data source
+keywords:
+  - grafana
+  - ecs
+  - enterprise
+  - aws
+  - marketplace
+  - activate
```

Comment:
These keywords don't look right, probably copy-paste from an existing file?

Reply:
removed

@jdbaldry wanted to see if this was the overall structure of how we would make the concept and tasks, so sorry for the imperfect PR 👯

Reply:
No worries at all :) thanks for coming back to restructure things!